internal/ingress/controller/nginx.go

```diff
@@ -1022,3 +1004,62 @@ func cleanTempNginxCfg() error {
 
 	return nil
 }
+
+type redirects struct {
```

Comment:
nitpick but this reads better as singular: `redirect` without `s`.

Reply:
done